configs/ner/bert_softmax/bert_softmax_cluener_18e.py

```diff
@@ -0,0 +1,79 @@
+_base_ = [
+    '../../_base_/schedules/schedule_adadelta_18e.py',
+    '../../_base_/default_runtime.py'
+]
+
+categories = [
+    'address', 'book', 'company', 'game', 'government', 'movie', 'name',
+    'organization', 'position', 'scene'
+]
+
+test_ann_file = 'data/cluener_public/dev.json'
+train_ann_file = 'data/cluener_public/train.json'
+vocab_file = 'data/cluener_public/vocab.txt'
+pretrained = 'checkpoints/bert_pretrain.pth'
```

Comment:
We can provide an url in `pretrained`, so that users do not have to download them manually.